Makefile

```diff
@@ -495,7 +512,8 @@ examples: $(EXAMPLE_BINS)
 
 py$(PROJECT): py
 
-py: $(PY$(PROJECT)_SO) $(PROTO_GEN_PY)
+py:	checks \
```

Comment:
If people only compile the python target then checks are ignored. Compiling with NCCL but not CUDA is valid I believe but doesn't work at runtime.